### ISSUE TITLE
.blazerc mentioned (wrong filename) on import error at bazel startup

### ISSUE THREAD

**Issue by peakschris** [bazelbuild/bazel#22685]
### Description of the bug:

If a .bazelrc file has a error due to failed import, the following message is displayed:
```
[FATAL 05:46:36.066 src/main/cpp/blaze.cc:1104] Unexpected error reading .blazerc file 'd:\workdir\javaproject2/out/bazelrc/imported.bazelrc':
```

The error should reference `.bazelrc` not `.blazerc`

### Which operating system are you running Bazel on?

windows

### What is the output of `bazel info release`?

7.2.0

**Comment by fmeum** [comment]
@bazel-io fork 7.3.0

**Comment by iancha1992** [comment]
A fix for this issue has been included in [Bazel 7.3.0 RC1](https://github.com/bazelbuild/bazel/releases/tag/7.3.0rc1). Please test out the release candidate and report any issues as soon as possible.
If you're using Bazelisk, you can point to the latest RC by setting `USE_BAZEL_VERSION=7.3.0rc1`. Thanks!
